sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/MediaStreamingAudio.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.models;
+
+import com.azure.communication.common.CommunicationIdentifier;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+
+/** The MediaStreamingAudio model. */
+public class MediaStreamingAudio extends MediaStreamingPackageBase {
+
+    /*
+     * The audio data.
+     */
+    private final String audioData;
+
+    /*
+     * The timestamp of when the media was sourced.
+     */
+    private final OffsetDateTime timestamp;
+
+    /*
+     * The participantId.
+     */
+    private final CommunicationIdentifier participant;
+
+    /*
+     * Indicates if the received audio buffer contains only silence.
+     */
+    private final boolean silent;
+
+    /**
+     * The constructor
+     *
+     * @param audioData The audio data.
+     * @param timestamp The timestamp of when the media was sourced.
+     * @param participant The participantId.
+     * @param silent Indicates if the received audio buffer contains only silence.
+     */
+    MediaStreamingAudio(String audioData, String timestamp, CommunicationIdentifier participant, boolean silent) {
+        this.audioData = audioData;
+        this.timestamp = OffsetDateTime.parse(timestamp, DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+        this.participant = participant;
```

Comment:
The construction of the OffsetDateTime attribute, should be applied to the `participant` as well.